html/modules/custom/hr_paragraphs/templates/hr-paragraphs-iframe-tableau.html.twig

```diff
@@ -0,0 +1,32 @@
+<div{{ create_attribute()
+  .setAttribute('data-width', width)
+  .setAttribute('data-height', height)
+}}>
+  <object class='tableauViz' style='display:none;'>
+    <param name='host_url' value='https%3A%2F%2Fpublic.tableau.com%2F' />
+    <param name='embed_code_version' value='3' />
+    <param name='site_root' value='' />
+    <param name='name' value='{{ embed_id }}' />
+    <param name='tabs' value='no' />
+    <param name='toolbar' value='yes' />
+    <param name='animate_transition' value='yes' />
+    <param name='display_static_image' value='yes' />
+    <param name='display_spinner' value='yes' />
+    <param name='display_overlay' value='yes' />
+    <param name='display_count' value='yes' />
+    <param name='language' value='en-US' />
+  </object>
+
+  <script type='text/javascript'>
```

Comment:
You want me to clean this up and make sure a single script can instantiate all the paragraphs on a page?

Reply:
ok I never got back to it today, we don't have to.